lib/monitor/watch.js

```diff
@@ -195,6 +195,51 @@ function ignoredFileTypesForFind(dir) {
   return ' \\( ' + paths.join(' -or ') + ' \\) -or ';
 }
 
+// Inspired by http://underscorejs.org/docs/underscore.html
+function throttle(func, wait, options) {
```

Comment:
This didn't actually work at all. A throttle will always let the first event through, and then *throttle* the rest. What it should have been was a [debounce](https://remysharp.com/2010/07/21/throttling-function-calls).